code_sponsor/settings.py

```diff
@@ -40,12 +41,12 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
-ROOT_URLCONF = 'code_sponsor.urls'
+ROOT_URLCONF = env('ROOT_URLCONF', default='code_sponsor.urls')
 
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [os.path.join(BASE_DIR, 'templates')]
+        'DIRS': [BASE_DIR + "/templates"]
```

Comment:
You can probably change this to `BASE_DIR('templates')`

Reply:
Will do